testbot.py

```diff
@@ -160,4 +165,46 @@ async def join_game(arguments: list, message: discord.message):
     logging.info(f'Assigned {message.author} to {game_name}')
     await message.channel.send(f'Have fun playing {game_name}, {message.author}!')
 
+
+async def make_poll(message: discord.message):
+    command = message.content.replace("pb poll", '').strip()
+
+    if not command or "\"" not in command or ',' not in command:
```

Comment:
You're combining both the logging and the feedback for missing title and arguments into one. It would be more useful for the user to know what exactly he messed up on. Have one check for title, and another for arguments?